### ISSUE TITLE
Make how the FAIR scores are calculated clearer on the portals metrics tab

### ISSUE THREAD

**Issue by laijasmine** [NCEAS/metacatui#1820]
It was noted by a researcher looking at the [metrics tab](https://search.dataone.org/portals/mosaic/Metrics) in the portals that it wasn't clear how the FAIR scores were calculated:
> The DataONE FAIR assessment is unclear to me. I clicked on the link and I generally understand what the FAIR principles are, but how that equates to a score is not clear to me.  Perhaps the information is there at the linked page (I didn't look that hard!).

**Is your feature request related to a problem? Please describe.**
I think it might be unclear to individuals who either (1) might not have submitted data before or (2) haven't investigated the individual assessment reports to understand where the percentages are coming from and by what criteria we are evaluating the datasets by. 

The reader really needs to go to the [paper](https://www.nature.com/articles/sdata201618) or the [recorded webinar](https://www.dataone.org/webinars/quantifying-fair-metadata-improvement-and-guidance-dataone-repository-network/) for more information. This might be one level too deep for readers to find the information they need quickly.

Maybe directing them to finding the Assessment Report button on an individual dataset will help?

**Comment by laurenwalker** [comment]
Thanks Jasmine, I agree that we could be more transparent about how these scores are calculated. Listing the suite of checks with their descriptions would be a good start. @gothub @mbjones - Is there a way to grab a suite of check descriptions from the metadata quality service API?

**Comment by mbjones** [comment]
@laurenwalker yes, there is an API, although I'm not sure how much has been implemented. Docs are here: https://github.com/NCEAS/metadig-engine/blob/main/docs/api.md

For example, list all checks:
https://docker-ucsb-4.dataone.org:30443/quality/checks

And pull the XML definition for a specific check identifier check: 
https://docker-ucsb-4.dataone.org:30443/quality/checks/check.pub.date.1

None of this details "how" the scores are calculated, but we did review that in a few sets of slides which I can send to you if needed. The basic formula is:

score = (Rpass + Opass) / (Rpass + Rfail + Opass)

where R is required and O is optional checks. So, its basically the ratio of pass to fail, where failing optional checks doesn't count against you, but passing optional checks increases your score.

**Comment by laurenwalker** [comment]
Thanks Matt, that is helpful. I think just listing the checks and their descriptions like we do in the individual reports would be a good start

**Comment by mbjones** [comment]
Those are produced from the full XML report of the run for a dataset, which includes the list of the checks and their results. While this will differ from dataset to dataset, here's an example of what MetacatUI pull for a single dataset run:

https://docker-ucsb-4.dataone.org:30443/quality/runs/FAIR-suite-0.3.1/https://pasta.lternet.edu/package/metadata/eml/knb-lter-hfr/361/2?_=1625771789903
